### ISSUE TITLE
[documentation] more examples

### ISSUE THREAD

**Issue by Dinth** [custom-cards/flex-table-card#38]
Hey. Just wanted to raise a feature request regarding the documentation to include more real-life examples. Its really hard to understand how to use flex-table-card and how to make even simplest things with it, especially now when a large part of documentation is about features which were deprecated (attr_as_list).

**Comment by daringer** [comment]
yes, totally with you it's time to throw out the deprecated docs and add some new ones.+
Also feel free to check the closed issues for examples of how things are done and don't hesitate to ask here if you have something specific you would like to achieve

**Comment by maxwroc** [comment]
+1 for extending documentation

First of all I'm missing screenshots. It is way easier to find out what can be achieved with the card just by browsing screenshots. 

There should be at least couple (most awesome) examples on the main README.md file.
